cerealize/test/impl.cc

```diff
@@ -93,7 +96,7 @@ TEST(Cerealize, NamedJSON) {
 };
 
 TEST(Cerealize, BinarySerializesAndParses) {
-  FileSystem::CreateDirectory(FLAGS_cerealize_test_tmpdir, FileSystem::CreateDirectoryParameters::Silent);
+  FileSystem::CreateDir(FLAGS_cerealize_test_tmpdir, FileSystem::CreateDirParameters::Silent);
```

Comment:
:+1: good point. Here, unfortunately, there's a reason behind rename.

(Visual Studio is great for debugging, but development is still much more pleasant on Linux!)